scripts/extract.py

```diff
@@ -27,6 +27,7 @@ def __init__(self, arguments):
         self.output_dir = self.faces.output_dir
 
         self.export_face = True
+        self.save_interval = self.args.save_interval
```

Comment:
If extraction is being run as part of convert (i.e. no alignments file exists when someone runs convert) then `save_interval` won't exist in `arguments` and the code will error out, so remove this line and see below.

Reply:
I added the check here so we don't have to keep looking if the arg exists. I don't know if it matters much with performance but it's a tiny bit optimization I guess